app/javascript/components/time-hero.jsx

```diff
@@ -8,8 +8,9 @@ import { spacing } from '@lib/theme';
 import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
 
 const Wrapper = styled(SpacedRow)`
+  margin-top: ${spacing.m};
   align-items: flex-end;
-  margin-bottom: ${spacing.xxl};
+  margin-bottom: 26px;
```

Comment:
Does this orphan spacing.xxl?

Reply:
Nope, though it's only used in one other place -- fixed-content.jsx